gtfs-realtime/spec/en/reference.md

```diff
@@ -114,7 +129,7 @@ Note that the update can describe a trip that has already completed.To this end,
 |------------------|------------|-------------------|-------------------|
 | **trip** | [TripDescriptor](#message-tripdescriptor) | required | The Trip that this message applies to. There can be at most one TripUpdate entity for each actual trip instance. If there is none, that means there is no prediction information available. It does *not* mean that the trip is progressing according to schedule. |
 | **vehicle** | [VehicleDescriptor](#message-vehicledescriptor) | optional | Additional information on the vehicle that is serving this trip. |
-| **stop_time_update** | [StopTimeUpdate](#message-stoptimeupdate) | repeated | Updates to StopTimes for the trip (both future, i.e., predictions, and in some cases, past ones, i.e., those that already happened). The updates must be sorted by stop_sequence, and apply for all the following stops of the trip up to the next specified one. |
+| **stop_time_update** | [StopTimeUpdate](#message-stoptimeupdate) | required | Updates to StopTimes for the trip (both future, i.e., predictions, and in some cases, past ones, i.e., those that already happened). The updates must be sorted by stop_sequence, and apply for all the following stops of the trip up to the next specified one.  At least one stop_time_update must be provided for the trip. |
```

Comment:
When a trip is canceled and its `ScheduleRelationship` is reported as `CANCELED`, some (most?) operators don't list any stops in `StopTimeUpdate` as its assumed that the trip will not be making any stops.

Do we want this to be the case?  Or, would it preferred to list all stops with the `StopTimeUpdate`'s `ScheduleRelationship` as `SKIPPED` for all stops?